### ISSUE TITLE
Update Alignment Marks in 15.2.0 => PSRam error

### ISSUE THREAD

**Issue by nliaudat** [jomjol/AI-on-the-edge-device#2351]
### The Problem

When I try to update the Alignment Marks in 15.2.0 version, I've got PSRam error.
The image is empty (0B) and it's output a "[PSRAM] Shared memory in PSRAM too small (STBI)" error

### Version

15.2.0

### Logfile

```shell
[0d00h03m19s] 2023-04-26T20:29:05 <ERR> [PSRAM] Shared memory in PSRAM too small (STBI) to fit additional 921601bytes! Available: 339087 bytes!
[0d00h03m19s] 2023-04-26T20:29:05 <ERR> [C IMG BASIS] CImageBasis-image: Failed to load /sdcard/config/reference.jpg! Is it corrupted?
[0d00h03m19s] 2023-04-26T20:29:05 <ERR> [PSRAM] Failed to allocate 4294957360 bytes in PSRAM for 'c_align_and_cut_image->odata'!
[0d00h03m19s] 2023-04-26T20:29:06 <ERR> [C IMG BASIS] /sdcard/img_tmp/ref0_org_org.jpg is empty!
[0d00h03m21s] 2023-04-26T20:29:07 <ERR> [PSRAM] Shared memory in PSRAM too small (STBI) to fit additional 921601bytes! Available: 31872 bytes!
[0d00h03m21s] 2023-04-26T20:29:07 <ERR> [C IMG BASIS] CImageBasis-image: Failed to load /sdcard/config/reference.jpg! Is it corrupted?
[0d00h03m21s] 2023-04-26T20:29:07 <ERR> [PSRAM] Failed to allocate 4294957360 bytes in PSRAM for 'c_align_and_cut_image->odata'!
[0d00h03m21s] 2023-04-26T20:29:07 <ERR> [C IMG BASIS] /sdcard/img_tmp/ref0_org.jpg is empty!
```

### Additional Context

Tried in 2 differents esp-cam = same result. 

The updated reference image is 0b = corrupted

_No response_

**Comment by caco3** [comment]
Thanks for reporting this.
I can confirm it!

I will look into it, but it will take me some days.

Until then you could revert to 15.1.1 to update the Aligment Marks (and then upgrade again)

**Comment by jurasjo** [comment]
I can also confirm that. I thought it was about corrupted SD card. Now everything is clear.

**Comment by amor20git** [comment]
I have same problem since update

**Comment by turbolooser** [comment]
me2. just reverted to 15.1.1 and its working again

**Comment by caco3** [comment]
The issue got solved in 15.2.1
